wdae/wdae/conftest.py

```diff
@@ -77,32 +85,38 @@ def default_gene_models(global_gpf_instance):
     return global_gpf_instance.genomes_db.get_gene_models()
 
 
-@pytest.fixture(scope='function')
```

Comment:
Try to copy this fixture from dae module. Before running the tests remove all of the `*-eff` files.